Formula/perl.rb

```diff
@@ -18,9 +18,9 @@ class Perl < Formula
 
   patch do
     # Enable build support on macOS 11.x
-    # https://github.com/Perl/perl5/pull/17946
-    url "https://github.com/Perl/perl5/pull/17946.diff?full_index=1"
-    sha256 "677a6e63412a4cfd2dc1fe16ccca53d789be5ac1acfb5ebb3130c40b51e5e94a"
+    # Temporary until https://github.com/Perl/perl5/pull/17946 is merged
```

Comment:
I prefer to always start that comment with the literal word _Remove_ so it’s 1. unambiguously actionable, and 2. on the off chance it’s forgotten, it can be grepped.

What do you think?

Reply:
@claui Sounds good to me, feel free to change it 👍

Reply:
@BytesGuy Done.
Fyi, I also changed the Git revision of the formula patch to a plain commit revision.

Reply:
@claui Thank you 🎉